ta/pauth_test/ta_entry.c

```diff
@@ -0,0 +1,68 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
Any reason we need a new TA instead of adding this to the os_test TA

Reply:
I followed the same thing which I did for BTI. Can move it to os_test if you think that is the right place.

Reply:
Another idea could be to have a single TA for all ARM security extensions i.e combine the BTI and PAUTH together and later add MTE tests also to it.

Reply:
That's another option. I think there is a value in trying to keep the number of TAs down. So if it's not too inconvenient I'd rather have it all in the os_test TA.